src/components/WalletItem/index.tsx

```diff
@@ -44,7 +44,11 @@ export const WalletItem: React.FunctionComponent<Wallet> = (props: Wallet) => {
     return (
         <div style={style} className={cName}>
             <div className="cr-wallet-item__info">
-                {iconUrl ? (<img alt="" className="cr-wallet-item__image-icon" src={iconUrl} />) : (<CryptoIcon className="cr-wallet-item__icon" code={currency.toUpperCase()} />)}
+                {iconUrl ? (
+                    <span className="cr-crypto-icon cr-wallet-item__icon">
+                        <img alt={currency.toUpperCase()} src={iconUrl} />
+                    </span>
```

Comment:
Why did you add this span?
You can apply className attribute directly to img tag

Reply:
span exists in CryptoIcon component.
I wanted to modify less as possible for existing consistency.
I CAN remove span from both, SHOULD I?

Reply:
I see
No, don't do that